html/css/404.css

```
@@ -0,0 +1,84 @@
+.fundo{
```

Hình như phải như này mà c
```suggestion
.fundo {
```